android/src/ti/map/AnnotationProxy.java

```diff
@@ -351,34 +363,23 @@ public void onPropertyChanged(String name, Object value)
 			updateInfoWindow();
 		} else if (name.equals(TiC.PROPERTY_LEFT_BUTTON)) {
 			getOrCreateMapInfoWindow().setLeftOrRightPane(value, TiMapInfoWindow.LEFT_PANE);
-			if (value == null) {
-				Object leftView = getProperty(TiC.PROPERTY_LEFT_VIEW);
-				if (leftView != null) {
-					getOrCreateMapInfoWindow().setLeftOrRightPane(leftView, TiMapInfoWindow.LEFT_PANE);
-				}
-			}
```

Comment:
If you look a few lines above you'll notice that there's a check for value == null. If it passes, the function is cancelled (return;). This snippet never gets executed.